be/src/common/status.h

```diff
@@ -413,17 +424,6 @@ class Status {
 
     bool ok() const { return _code == ErrorCode::OK; }
 
-    bool is_io_error() const {
```

Comment:
are you sure we could remove is_io_error method?

Reply:
I have checked these error code in `is_io_error`, replace `IO_ERROR` to `TEST_FILE_ERROR`; removed `READ_UNENOUGH` as this error code is never used; replace `CHECKSUM_ERROR` and `FILE_DATA_ERROR` to `INTERNAL_ERROR` as such 2 error code are only used in meta header file.